### PR TITLE
Add support for hofff/contao-consent-bridge

### PR DESCRIPTION
**Pull request by dmolineus**
This pull request provides support the the hofff/contao-consent-bridge as requested in #12. The implementation doesn't effect the bundle if the consent bridge is not installed.

### CODE REVIEW

File: src/ContaoManager/Plugin.php [hunk 1]
```diff
@@ -13,6 +13,7 @@
 use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
 use Contao\ManagerPlugin\Bundle\Parser\ParserInterface;
 use HeimrichHannot\GoogleMapsBundle\HeimrichHannotGoogleMapsBundle;
+use Hofff\Contao\Consent\Bridge\HofffContaoConsentBridgeBundle;
```
Comment (Defcon0) [inline src/ContaoManager/Plugin.php:16]: Won't that throw an exception if the bundle is not installed? Did you test that case sufficiently?

File: src/DependencyInjection/GoogleMapsExtension.php [hunk 2]
```diff
@@ -8,11 +8,15 @@
...
 namespace HeimrichHannot\GoogleMapsBundle\DependencyInjection;
 
+use HeimrichHannot\GoogleMapsBundle\EventListener\ConsentBridgeListener;
+use Hofff\Contao\Consent\Bridge\HofffContaoConsentBridgeBundle;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
+use function class_exists;
```
Comment (Defcon0) [inline src/DependencyInjection/GoogleMapsExtension.php:18]: Why is that necessary? In the core a backslash is used (`\class_exists`). In fact we could use the function without any `use` statement at all, no?

**Review by Defcon0 — CHANGES_REQUESTED** [review]

File: src/ContaoManager/Plugin.php (hunk 1 above)
Comment (dmolineus) [reply src/ContaoManager/Plugin.php:16]: No, you can use `::class` for classes that does not exist, see https://www.php.net/manual/de/language.oop5.constants.php#121613 or https://3v4l.org/REY04#v7.0.33

Comment (Defcon0) [reply src/ContaoManager/Plugin.php:16]: OK, then I guess this is also true if you use a `use` statement instead of the fully qualified class name insteadt?

File: src/DependencyInjection/GoogleMapsExtension.php (hunk 2 above)
Comment (dmolineus) [reply src/DependencyInjection/GoogleMapsExtension.php:18]: It's recommend not using global functions to optimize performance (see https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/). You may use the backslash or an explicit `use function` statement. It's a matter of taste what to use.

Comment (Defcon0) [reply src/DependencyInjection/GoogleMapsExtension.php:18]: OK, then we should go with the way it's done in the core which is the backslash version.

File: src/ContaoManager/Plugin.php (hunk 1 above)
Comment (dmolineus) [reply src/ContaoManager/Plugin.php:16]: Yes, it only imports the name and does not try to autoload the class. See https://3v4l.org/erFYN

**Comment by dmolineus** [comment]
I switched to the backslash notation for global functions.

**Comment by Defcon0** [comment]
New version is there: 2.5.0 :+1: 

Thanks!
